chef/automate
# Infra api documentation

Signed-off-by: Kallol Roy <karoy@progress.com>

### :nut_and_bolt: Description: What code changed, and why?

Infra Server has added APIs to get the server details, org details and the objects.



### :chains: Related Resources
https://chefio.atlassian.net/browse/STALWART-72

### :+1: Definition of Done
- The API document should be visible in netlify preview page

### :athletic_shoe: How to Build and Test the Change
- cd components/docs-chef-io
- make serve
- check the document in http://localhost:1313/automate/api/

### :white_check_mark: Checklist

**All PRs** must tick these:


With occasional exceptions, all PRs **from Progress employees** must tick these:


All PRs **from Progress employees** should tick these if appropriate:


*Please add a note next to any checkbox above if you are NOT ticking it.*

### :camera: Screenshots, if applicable
